src/pilot/assign_role.py

```diff
@@ -274,26 +274,24 @@ def define_target_raid_config(role, drac_client):
 def get_raid_controller_id(drac_client):
```

Comment:
Would you mind renaming this method to get_raid_controller_ids()?

Reply:
Done.